### PR TITLE
 tests: Reorder tests and move most of extended tests up to normal tests

### PR DESCRIPTION
**Pull request by ken2812221**
The travis should run almost all jobs unless it takes really long time, however it does not take too long for now. So it's time for moving it to normal job.

(The test sort is to see how many conflict will this cause, will drop it if there are too many)

The first commit can be reviewed by `git diff --color-moved=plain`

### CODE REVIEW

**Comment by MarcoFalke** [comment]
utACK 6e4d6ed487fcc65f551cdefc212aca329b6f70cd

**Comment by ken2812221** [comment]
Update: It seems impossible to run feature_pruning on travis, so I re-disable it.

File: test/functional/test_runner.py [hunk 1]
```diff
@@ -58,102 +58,115 @@
...
-    # vv Tests less than 5m vv
-    'feature_block.py',
-    'rpc_fundrawtransaction.py',
-    'p2p_compactblocks.py',
-    'feature_segwit.py',
+    'feature_fee_estimation.py',
     # vv Tests less than 2m vv
-    'wallet_basic.py',
-    'wallet_labels.py',
+    'feature_block.py',
```
Comment (MarcoFalke) [inline test/functional/test_runner.py]: to be honest, I don't think the reordering matters. for example, feature_block was the third test previously and is now the second test. I think it is most important to have the longest running test as the first test. The order of the remaining ones shouldn't matter much if they are reasonably sorted by run time.

**Review by conscott — COMMENTED** [review]
Tested ACK d641c29a5a7dcbf186e32af2e22cd3601e5c49fb

**Pull request merged**